src/Identity/Extensions.Core/src/PasswordHasher.cs

```diff
@@ -65,6 +65,26 @@ public PasswordHasher(IOptions<PasswordHasherOptions> optionsAccessor = null)
             _rng = options.Rng;
         }
 
+        // Compares two byte arrays for equality. The method is specifically written so that the loop is not optimized.
```

Comment:
Would recommend wrapping this in `#if NETSTANDARD2_0` to avoid usage in netcoreapp3.0, where there is a better API.